src/Process/ProcessUtils.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace GrumPHP\Process;
+
+/**
+ * ProcessUtils is a bunch of utility methods.
+ *
+ * This class contains static methods only and is not meant to be instantiated.
+ *
+ * @author Martin Hasoň <martin.hason@gmail.com>
+ */
+class ProcessUtils
+{
+    /**
+     * Escapes a string to be used as a shell argument.
+     * Adapted from the Symfony package.
+     *
+     * @param string $argument The argument that will be escaped
+     *
+     * @return string The escaped argument
+     *
+     * @see \Symfony\Component\Process\ProcessUtils::escapeArgument()
+     */
+    public static function escapeArgument($argument)
```

Comment:
Wouldn't it be better to use the new implementation in Symfony 4 and move that logic to this util class?
See : https://github.com/symfony/process/blob/4.0/Process.php#L1515-L1535

Reply:
Can you also add some test cases for the escapeArgument method to make sure it works?